cuenca_validations/types/general.py

```diff
@@ -9,10 +15,39 @@ def __init__(self, *args, **kwargs):
         sanitize_dict(self)
 
 
+class CJSONEncoder(json.JSONEncoder):
+    def default(self, o):
+        try:
+            encoded = sanitize_item(o)
+        except AttributeError:
+            encoded = super().default(o)
+        return encoded
+
+
 class StrictPositiveInt(StrictInt, PositiveInt):
     """
     - StrictInt: ensures a float isn't passed in by accident
     - PositiveInt: ensures the value is above 0
     """
 
     ...
+
+
+def digits(
+    min_length: Optional[int] = None, max_length: Optional[int] = None
+) -> Type[str]:
+    namespace = dict(min_length=min_length, max_length=max_length)
+    return type('DigitsValue', (Digits,), namespace)
+
+
+class Digits(ConstrainedStr):
+    @classmethod
+    def validate_digits(cls, value: str) -> str:
+        if not value.isdigit():
+            raise NotDigitError
```

Comment:
If this doesn't return a useful error message, you should issue a PR to Pydantic and redefine it in `cueca_validations.errors` in the meantime

Reply:
The close mi issue: https://github.com/samuelcolvin/pydantic/issues/1675 so I just redefined the message.